src/state/selectors/index.js

```diff
@@ -0,0 +1,22 @@
+
+/**
+* Return the manifest that belongs to a certain window.
+* @param {object} state
+* @param {String} windowId
+* @return {object}
+*/
+export function getWindowManifest(state, windowId) {
```

Comment:
I think it might make more sense for state to have a `registry `of manifests in store with props including current window id(s) and manifest values denormalized.  The lookup would filter for a manifest v5 uuid.  It could look like this:

The selector simply reduces using a uuidv5 (e.g. xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx) created from the manifest uri as the filter.
```
{
  "manifests": {
    "xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx": {
      "manifest": "http://some.manifest",
      "windows": [],
      "canvases": [],
      "iiifProperty": "String, etc."
    }
  }
}
```

Reply:
Would this be to avoid the state digging? I can see some advantages to this for request reasons. In terms of this particular lookup, though, wouldn't we then need to loop over all the manifests to get the answer instead of using the id the window has attached to itself on creation? Honestly I can see having multiple selectors for each of these purposes.

I think the PR as a whole is good for now, given the way the manfiestLogo component is implemented above, and for setting up a selector pattern to let us call on the store a bit less.

Reply:
There is no loop, it works like a map, you basically just select the manifest object by index.  

I would suggest a HoC `withManifest(WrappedComponent)`, that gets and sets a `manifestId `prop for a component.  Then, the component can get whatever values it wants from store registry that relates to the manifest using a selector passing the manifest `uuid `as the key.  

The strategy is to decouple window state from the content as much as possible.  The window should just do its job of being a window and doing window actions and not try to manage its content props.